.github/workflows/lint.yaml

```diff
@@ -0,0 +1,29 @@
+name: Lint
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.10.2'
+
+    - name: Install packages
+      run: |
+        pip install .
+        pip install -r requirements-dev.txt
```

Comment:
There is (currently) no requirements-dev.txt in this PR, so this is failing the linter.